ci-operator/step-registry/cucushift/installer/rehearse/aws/c2s/ipi/disconnected/private/deprovision/cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private-deprovision-chain.yaml

```diff
@@ -1,6 +1,7 @@
 chain:
   as: cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private-deprovision
   steps:
+  - ref: gather-log-bundle-from-private-cluster
```

Comment:
How about rename it `gather-installer-boostrap-logs-in-bastion`

Reply:
updated.